README.md

```diff
@@ -8,13 +8,65 @@ View the [demo](https://adobedocs.github.io/dev-site-documentation-template/) ru
 
 The slack channel #adobeio-onsite-onboarding is our main point of contact for help. Feel free to join the channel and ask any questions. 
 
-## How to develop
+## Prerequisite: Install Yarn 4
+
+This site uses Yarn 4 to manage package dependencies. If you do not have Yarn 4 installed, follow these instructions:
+
+1. Install [Node.js LTS](https://nodejs.org/en/download/). At the time of writing, the latest LTS version is `v20.12.2`.
+2. Install Yarn 4 by running the following command:
+
+   ```bash
+   corepack enable
+   ```
+
+Corepack is a binary shipped with all Node.js releases starting from 16.10.
+
+3. Update Yarn to the latest version by running the following command:
+
+   ```bash
+   yarn set version stable
+   ```
+
+4. Make sure you are now using version 4.1.1 of Yarn:.
+
+   ```bash
+   yarn --version
+   ```
+
+That's it. For more information on Yarn 4 usage, see [Yarn usage](https://yarnpkg.com/getting-started/usage).
+
+For the documentation developer, please read these sections on how to:
+
+- [Arrange the structure content of your docs](https://github.com/adobe/gatsby-theme-aio#content-structure)
+- [Linking to pages](https://github.com/adobe/gatsby-theme-aio#links)
+- [Using assets](https://github.com/adobe/gatsby-theme-aio#assets)
+- [Setting Global Navigation](https://github.com/adobe/gatsby-theme-aio#global-navigation)
+- [Setting Side Navigation](https://github.com/adobe/gatsby-theme-aio#side-navigation)
+- [Using content blocks](https://github.com/adobe/gatsby-theme-aio#jsx-blocks)
+- [Notes on using Markdown](https://github.com/adobe/gatsby-theme-aio#writing-enhanced-markdown)
+
+## How to build the site locally
+
+After installing `yarn` v4 as noted above, you can develop locally by running the following commands:
+
+1. Install the dependencies:
+
+   ```bash
+   yarn install
+   ```
+
+1. Run a development build:
```

Comment:
If you take a look at `package.json`, it already has commands that are wrappers around `yarn`. Do you think it makes sense to expose `yarn` commands separately and use them instead of the commands specified in `package.json`?

Reply:
Yarn does run the scripts from package.json 

For instance `yarn dev` runs the `"dev": ...` script, `yarn build` runs `"build": ...` and so on 

You can confirm by running `yarn dev --help`, first line will be `gatsby develop` which is the script command